File: app/src/main/java/com/govnokoder/velotracker/messages/SharedViewModel.java
package com.govnokoder.velotracker.messages;

import android.location.Location;

import androidx.lifecycle.MutableLiveData;
import androidx.lifecycle.ViewModel;

import com.govnokoder.velotracker.BL.ParcelableTraining;

public class SharedViewModel extends ViewModel {
    public MutableLiveData<ParcelableTraining> messagesParcelableTraining = new MutableLiveData<ParcelableTraining>();
    public void sendMessage(ParcelableTraining parcelableTraining){
        messagesParcelableTraining.setValue(parcelableTraining);
    }
}

File: app/src/main/java/com/coursework/velotracker/Messages/SharedViewModel.kt
package com.coursework.velotracker.Messages

import androidx.lifecycle.MutableLiveData
import androidx.lifecycle.ViewModel
import com.coursework.velotracker.BL.Model.Training.ParcelableTraining

class SharedViewModel(): ViewModel() {
    var parcelableTraining:MutableLiveData<ParcelableTraining> = MutableLiveData<ParcelableTraining>()
    fun sendMessage(parcelableTraining: ParcelableTraining){
        this.parcelableTraining.value = parcelableTraining
    }
}